webview/player/player.css

```
@@ -56,10 +56,12 @@ video {
 
 .progress-bar {
   display: flex;
+  align-items: center;
   background-color: transparent;
   position: absolute;
-  bottom: 3.24px;
+  bottom: 0;
```

The bottom value of -2 places the lifted progress bar at the right place estimately.
```suggestion
  bottom: -2px;
```